docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Environment Variables"
+---
+
+Environment variables are useful when:
+
+- Values are different across developer machines.
+- Values are different across multiple environments: (dev, staging, qa, prod)
+- Values change frequently and are highly dynamic.
+
+Environment variables can be changed easily - especially when running in CI.
+
+Oryx uses [dotenv](https://github.com/motdotla/dotenv) for environment variables management. 
+
+## Storefront Variables 
+
+The list of environment variables to configure Storefront:
+
+| Variable | Required | Default | Description                               |
+|----------|----------|---------|-------------------------------------------|
+| `SCOS_BASE_URL` | Yes | None | Spryker GLUE API URL |
+| `STORE` | Yes | None | Default store name, e.g. DE, AT etc. The value is case-sensitive |
+| `PRICE_MODE` | No | None | Can be used to initialise the application with default price mode. When this variable is not provided, the backend will set the default one |
+| `ORYX_PRESET` | No | `b2c` | Used to set a preset name, available options: `b2b`, `b2c` |
+| `ORYX_FEATURE_VERSION` | No | `latest` | Used to set a feature level, available options: `latest`, `1.0`, `1.1`, etc. Only major and minor versions can be used |
+| `ORYX_LABS` | No | `false` | Turns on/off oryx labs features. These features are in experimental state and are not production ready |
```

Comment:
We use this as an internal env var to setup our environments, but it's not something our customers need to know about.

Reply:
Removed all 3 of them.